src/window/building_info.c

```diff
@@ -1253,7 +1253,6 @@ void window_building_info_depot_toggle_condition_type(void)
 void window_building_info_depot_toggle_condition_threshold(void)
 {
     building *b = building_get(context.building_id);
-    int step = config_get(CONFIG_GP_STORAGE_INCREMENT_4) ? 4 : 8;
```

Comment:
I think you removed this line by accident, please add it back

Reply:
Step is declared but never used...